src/sql/workbench/contrib/notebook/browser/notebook.css

```diff
@@ -82,6 +82,9 @@
 .notebookEditor .in-preview .actions-container .action-item:last-child {
 	margin-right: 14px;
 }
+.notebookEditor .in-preview .actions-container .action-item:last-child :not(.masked-icon) {
+	margin-left: 8px;
```

Comment:
So why is this necessary only for the non-masked icons?

Is this something generic to any icon we might add in the future?

Reply:
I want us to use the masked-icon process as it's easier to swap out icon colors for each theme. I did this with all the icons I updated in the notebook UI. The Manage Packages icon and its corresponding action comes in through a different route - Jupyter command - where I am unable to modify. 

I am at the moment trying to inject this `masked-icon` class into the workflow prior to the action item being appended t the action menu.